azure-servicebus/src/main/java/com/microsoft/azure/servicebus/MessageBody.java

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.azure.servicebus;
+
+import java.util.List;
+
+/**
+ * This class encapsulates the body of a message. Body types map to AMQP message body types.
+ * It has getters and setters for multiple body types.
+ * Client should test for body type before calling corresponding get method.
+ * Get methods not corresponding to the type of the body return null.
+ */
+public class MessageBody {
+    private MessageBodyType bodyType;
+    private Object valueData;
+    private List<List<Object>> sequenceData;
+    private List<byte[]> binaryData;
+    
+    private MessageBody() {}
+    
+    /**
+     * Creates message body of AMQPValue type.
+     * @param value AMQPValue content of the message. It must be of a type supported by AMQP.
+     * @return MessageBody instance wrapping around the value data.
+     */
+    public static MessageBody fromValueData(Object value)
+    {
+    	if(value == null)
+    	{
+    		throw new IllegalArgumentException("Value data is null.");
+    	}
+    	
+    	MessageBody body = new MessageBody();
+    	body.bodyType = MessageBodyType.VALUE;
+    	body.valueData = value;
+    	body.sequenceData = null;
+    	body.binaryData = null;
+    	return body;
+    }
+    
+    /**
+     * Creates a message body from a list of AMQPSequence sections.Each AMQPSequence section is in turn a list of objects.
+     * Please note that this version of the SDK supports only one AMQPSequence section in a message. It means only a list of exactly one sequence in it is accepted as message body.
+     * @param sequenceData a list of AMQPSequence sections. Each AMQPSequence section is in turn a list of objects. Every object in each list must of a type supported by AMQP.
+     * @return MessageBody instance wrapping around the sequence data.
+     */
+    public static MessageBody fromSequenceData(List<List<Object>> sequenceData)
```

Comment:
>fromSequenceData(List<List<Object>> sequenceData) [](start = 30, length = 49)

I wonder if its better to just expose `fromSequenceData(List<Object> sequenceData)` now.

Later whenever we decide to support the list of sequence, we could expose another overload which can accept list of list. `fromSequenceDataList(List<List<Object>> sequenceDataList)`